core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -167,7 +167,7 @@ class ReplicaManagerTest {
         .foreach(checkpointFile => assertTrue(Files.exists(checkpointFile),
           s"checkpoint file does not exist at $checkpointFile"))
     } finally {
-      rm.shutdown(checkpointHW = false)
+      CoreUtils.swallow(rm.shutdown(checkpointHW = false), this)
```

Comment:
Why is the exception swallowed here and all other places with the latest change? I did not mean to suggest this in my review. I clarified that in another [comment](https://github.com/apache/kafka/pull/15077#discussion_r1437341409).